### ISSUE TITLE
Support background transparency from emacs 29 (alpha-background)

### ISSUE THREAD

**Issue by BajaCali** [d12frosted/homebrew-emacs-plus#541]
Hello,

since Emacs 29 there's an option for setting _true_ background transparency. See [this article](https://kristofferbalintona.me/posts/202206071000/). It seems like I have to have emacs with caira installed, but I did not find option to do that alongside emacs plus.

Can you help me out with background transparency please?

**Comment by d12frosted** [comment]
As far as I can tell from the article, one needs to build with GTK3 **and** cairo support. The former is not something you want on macOS. I think Mitsuharu `emacs-mac` supports transparent frames, but someone needs to investigate what exactly needs to be ported to GNU Emacs in order to support transparency on macOS. If it's a trivial patch, we can include that as part of Emacs+. Though the better approach would be to send a PR to Emacs itself.

**Comment by jerkotak** [comment]
[This reddit thread](https://www.reddit.com/r/emacs/comments/v72tu6/comment/ibkqo06/?utm_source=share&utm_medium=web2x&context=3) suggests that GTK3 might not be necessary for `alpha-background` to work. However, I'm not sure how to build Emacs 29 with cairo support to check it. Is it possible?

**Comment by ypytomets** [comment]
May be related
- https://github.com/aaratha/homebrew-emacs-plus
- https://lists.gnu.org/archive/html/emacs-devel/2024-06/msg01007.html
- https://debbugs.gnu.org/cgi/bugreport.cgi?bug=71850

tl;dr: there's a patch https://debbugs.gnu.org/cgi/bugreport.cgi?filename=0001-Add-alpha_background-feature-for-NS-MacOS.patch;msg=22;att=1;bug=71850
